src/diffTool/diffProcessor.ts

```diff
@@ -14,18 +14,37 @@ import { applyRules } from "./rulesProcessor";
 
 export const diffRulesPath = path.join(__dirname, "../../resources/diff/rules");
 
+/**
+ * Generate differences between two RAML files and apply rules
+ * @param leftRaml - Base RAML file to compare
+ * @param rightRaml - Other RAML file to compare with the left RAML to find differences
+ * @param rulesPath - Optional rules file to apply on the differences
+ *
+ * @returns Changes to the APIs
+ */
+export async function findApiChanges(
+  leftRaml: string,
+  rightRaml: string,
+  rulesPath?: string
+): Promise<NodeDiff[]> {
+  const diffs = await diffRaml(leftRaml, rightRaml);
+  if (rulesPath == null) {
+    ramlToolLogger.info("Applying default rules on the differences");
+    rulesPath = path.join(diffRulesPath, "defaultRules.json");
```

Comment:
`diffRulesPath` is a constant, and is only used here. Make the filename part of that so you don't have to recompute every time.